cluster-autoscaler/vendor/modules.txt

```diff
@@ -254,7 +254,7 @@ github.com/go-ozzo/ozzo-validation
 github.com/go-ozzo/ozzo-validation/is
 # github.com/godbus/dbus/v5 v5.0.3 => github.com/godbus/dbus/v5 v5.0.3
 github.com/godbus/dbus/v5
-# github.com/gogo/protobuf v1.3.1 => github.com/gogo/protobuf v1.3.1
+# github.com/gogo/protobuf v1.3.2 => github.com/gogo/protobuf v1.3.1
```

Comment:
We need to be on v1.3.2, need to remove or update the replacement statement in `go.mod` for this one

Reply:
i'll update it